ext/cl_khr_icd.asciidoc

```diff
@@ -60,6 +63,14 @@ members can be appended.
 Functions which do not have an argument from which the vendor implementation
 may be inferred have been deprecated and may be ignored.
 
+*New in version 2.0.0*, object that are ICD 2 compatible should also contain
+a `disp_data` field of type `void *` that the loader is free to modify for a
+platform through the `clSetPlatformDispatchDataKHR` new API entry point
+In order to distinguish ICD 1 and ICD 2 objects, the pointer to
+`clGetPlatformIDs` in the dispatch structure must be set to
+`CL_ICD2_TAG_KHR`. The vendor driver is responsible for propagating the
```

Comment:
OK, so loader sets it to some objects, and then when creating new objects that are related to another object (cl_context), the driver is assumed to copy it by default? Could we make it more consistent by forcing the loader to manage this data always? It's supposed to be loader-specific data, not driver specific, right? And it's always in a well-specified position in the beginning of the CL objects?

Reply:
OK, found the explanation in this discussion: https://github.com/KhronosGroup/OpenCL-Docs/issues/1003 (the extensions of which obj args the loader doesn't know about). Could you perhaps move some of these justifications and examples to the proposal text to make it more self-contained?

Reply:
You are correct! I will add this to the rationale.

Reply:
Added in the issues section.